src/routers/transactions/products.ts

```diff
@@ -8,14 +8,96 @@ import { StatusCode } from '@/models/statusCodes';
 
 const router = createRouter();
 
+/**
+ * @api {get} /transactions/products/<type>
+ * @description Returns all product transactions of a specified type
+ *              and all their associated attributes.
+ *              Guaranteed fields are:
+ *                - productTransactionUuid
+ *                - datetime
+ *                - toPersonUuid
+ *                - fromPersonUuid
+ *                - amountOfProduct
+ *                - productUnits
+ *                - costPerUnit
+ *                - currency
+ *              Parameters have no effect on this request.
+ *              Associated attributes can be checked via /products/<type>
+ * @apiName GetAllProductTypeTransactions
+ *
+ * @apiError (401) Unauthorized - Must be admin
+ *
+ * @apiSuccess (200) {String} Successfully retrieved all <type> transactions
+ * @apiSuccessExample Success-Response:
+  [
+    {
+      "productTransactionUuid": "3cad5e7c-5444-4de1-aa81-a7d15acb35f1",
+      "datetime": "2018-01-23 04:05:06",
```

Comment:
doesn't utc have a specific format? doesn't this need a z at the end? that way clients will know that the datetime they are receiving is for sure UTC

Reply:
@Right2Drive is requesting that we add milliseconds to the datetime as well

Reply:
I feel like milliseconds is not useful for this application. But we can add a timezone, it should always be UTC.